blog_module/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from . import views
+
+app_name = 'blog_module'
+
+urlpatterns = [
+    path('', views.index, name='index'),
+    path('blogs/<slug:blog_id>/', views.detail, name='detail'),
```

Comment:
This will render out to `.../blog_module/blogs/<id>`

the URL could be shortened, but that's an opinion on taste.